<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0"> <class>IncidentForm</class> <widget class="QDialog" name="IncidentForm"> <property name="windowTitle"> <string>Инцидент (Редактирование происшествия)</string> </property> <layout class="QFormLayout"> <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Тип происшествия:</string></property></widget></item> <item row="0" column="1"><widget class="QComboBox" name="comboType"><item><property name="text"><string>Бедствие</string></property></item><item><property name="text"><string>Тренировка</string></property></item></widget></item> <item row="1" column="0"><widget class="QLabel"><property name="text"><string>Широта:</string></property></widget></item> <item row="1" column="1"><widget class="QLineEdit" name="editLat"></widget></item> <item row="2" column="0"><widget class="QLabel"><property name="text"><string>Долгота:</string></property></widget></item> <item row="2" column="1"><widget class="QLineEdit" name="editLon"></widget></item> <item row="3" column="0"><widget class="QLabel"><property name="text"><string>Описание:</string></property></widget></item> <item row="3" column="1"><widget class="QTextEdit" name="textDescription"></widget></item> <item row="4" column="1"><widget class="QPushButton" name="buttonSave"><property name="text"><string>Сохранить</string></property></widget></item> </layout> </widget> <resources> <connections> </connections></resources></ui>
